### ISSUE TITLE
Error Encountered by Cron Process Due to Missing settings.json File

### ISSUE THREAD

**Issue by AntoineKM** [towfiqi/serpbear#211]
**Issue Description:**
When running the `npm run start:all` command in the project, the `cron` process encounters an error related to a missing file or directory: `settings.json`. This error disrupts the execution of the `cron` task.

**Error Message:**
```
node:internal/process/promises:288
        triggerUncaughtException(err, true /* fromPromise */);
        ^
[Error: ENOENT: no such file or directory, open './data/settings.json'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: './data/settings.json'
}
```

**Steps to Reproduce:**
https://docs.serpbear.com/deployment/running-serpbear-locally

**Expected Behavior:**
The `cron` task should run without errors and successfully access the `settings.json` file.

**Additional Context:**
- Project version: 2.0.2
- Node.js version: 18.19.0

**Possible Solution:**
Ensure that the `settings.json` file exists in the specified path (./data/). If it does not exist, create the file with the required configuration.

**Comment by towfiqi** [comment]
The error should appear only once when the app tries to create settings.json file. Once the settings.json file has been created, you should not see the error message.

**Comment by AntoineKM** [comment]
Yes, I've seen that, but for new users like me it can be confusing, especially since I really followed the documentation step by step, it might be better to check if it exists first?

**Comment by towfiqi** [comment]
It already checks if the settings file exists or not. See `getAppSettings` function `cron.js` file.
